CONTRIBUTING.md

```diff
@@ -153,6 +153,18 @@ rather than conform to specific API names or argument patterns in the spec.
 
 For a deeper discussion, see: https://github.com/open-telemetry/opentelemetry-specification/issues/165
 
+## Running Tests Locally
+
+1. Go to your Contrib repo directory. `cd ~/git/opentelemetry-python-contrib`.
+2. Create a virtual env in your Contrib repo directory. `python3 -m venv my_test_venv`.
+3. Activate your virtual env. `source my_test_venv/bin/activate`.
+4. Clone the [OpenTelemetry Python](https://github.com/open-telemetry/opentelemetry-python) Python Core repo to a folder named `opentelemetry-python-core`. `git clone https://github.com/open-telemetry/opentelemetry-python.git opentelemetry-python-core`.
+5. Change directory to the repo that was just cloned. `cd opentelemetry-python-core`.
+6. Move the head of this repo to the hash you want your tests to use. This is currently the SHA `1a12fa0d681e37c1fda9cb8d46212ff3bbf6b76a` as seen in `.github/workflows/test.yml`. Use `git fetch && git checkout 1a12fa0d681e37c1fda9cb8d46212ff3bbf6b76a`.
+7. Go back to the root directory. `cd ../`.
+8. Make sure you have `tox` installed. `pip install tox`.
```

Comment:
Better to recommend installing `dev-requirements.txt` file

Reply:
Currently the `dev-requirements.txt` file does not have `tox` specified in it. So should `tox` be included with some specific version in the `dev-requirements.txt` file, or should it mentioned separately?

Reply:
I see. That depends on whether tox internally uses dev-requirements. So if tox uses dev-requirements to install these packages in every .tox/<env>, then adding tox to this file does not make sense. In that case probably we don't need to mention creation of virtualenv above and just ask the user to install tox and leave it to the user how they want to install Python CLI tools. We should probably mention specific version of tox as well.

If tox does not use dev-dependencies and they exist to enable the "manual" use case then I think we can add tox to it.

Reply:
According to me, tox uses dev-requirements and hence installing and running tox would be enough